src/map/script.cpp

```
@@ -7082,7 +7082,8 @@ static int script_countitem_sub(struct item *items, std::shared_ptr<item_data> i
 					continue;
 			}
 
-			count += items[i].amount;
+			if ((rental && itm->expire_time > 0) || (!rental && itm->expire_time == 0))
```

Maybe move it up, because these checks allow failing faster.